src/grpc/hpr_hotspot_service.erl

```diff
@@ -15,15 +15,13 @@ init(Stream) ->
     Stream.
 
 -spec send_packet(hpr_packet_up:packet(), grpcbox_stream:t()) ->
-    ok
-    | {ok, grpcbox_stream:t()}
-    | {ok, packet_router_pb:packet_router_packet_down_v1_pb(), grpcbox_stream:t()}
-    | {stop, grpcbox_stream:t()}
-    | {stop, packet_router_pb:packet_router_packet_down_v1_pb(), grpcbox_stream:t()}
-    | grpcbox_stream:grpc_error_response().
-send_packet(_PacketUp, Stream) ->
-    {ok, Stream}.
+    ok | {continue, grpcbox_stream:t()} | grpcbox_stream:grpc_error_response().
+send_packet(PacketUp, Stream) ->
+    _ = erlang:spawn(hpr_routing, handle_packet, [PacketUp, Stream]),
```

Comment:
I think this might be hard to debug if something goes seriously wrong in hpr_routing:handle_packet/2. Maybe use `proc_lib:spawn` instead? That generates a crash report if the process exits abnormally (exit reason is not `normal`, `shutdown`, or `{shutdown, Term}`.